src/components/modules/ContentLoader/ContentLoader.view.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="content-loader">
+    <slot />
+    <div
+      v-if="isLoading"
+      data-test="endpass-app-loader"
+      class="content-loader-overlay"
+    >
+      <span class="content-loader-label">
+        <slot name="label" />
+      </span>
+      <v-progress-circle
+        progress="25"
+        line-thickness="1"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import VProgressCircle from '@endpass/ui/kit/VProgressCircle';
+
+export default {
+  name: 'ContentLoaderView',
+
+  props: {
+    isLoading: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  components: {
+    VProgressCircle,
+  },
+};
+</script>
+
+<style lang="scss">
+.content-loader-overlay {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.6);
+  z-index: 100;
```

Comment:
Add `width: 100%; height: 100%` for more obvious behavior in other components

Reply:
it can't be with defined left,right,top,bottom values. should be only widht/height or left,right,top,bottom.
better to use last